### ISSUE TITLE
Clear, useable repository

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutSPACEs#41]
### Summary

Make repository README and docs clear for new users. Should be possible to start working on the project as a beginner to the project just from the information available in the repository. Should also guide users who are unfamiliar with Django/new to GitHub. 

_This issue concerns the ease of use of the **AutSPACEs repository** not the platform._

### What needs to be done?

- [ ] Clear READMEs
- [ ] Step-by-step instructions
- [ ] Community testing and feedback

### Who can help?

Everyone! Please test this repository and if anything is unclear or you are struggling with it, add a comment to this issue so w can improve thee repository. 

### How can I help

- [x] See if you are able to get the website running locally on your computer. If so, comment below that you were succesful; if not let us know where you got stuck
- [ ] Comment on this issue to tell us if you have any accessibility requirements or preferences the repo doesn't cater for so we can fix it to make it more accesible. 
- [ ] Comment on this issue to tell us if anything was confusing or could be clearer. Also let us know if you especially liked something and you don't want it changed!
